webpack/components/extensions/HostDetails/Tabs/ErrataTab.js

```diff
@@ -29,9 +31,13 @@ import './ErrataTab.scss';
 
 export const ErrataTab = () => {
   const hostDetails = useSelector(state => selectAPIResponse(state, 'HOST_DETAILS'));
-  const { id: hostId } = hostDetails;
+  const {
+    id: hostId,
+    content_facet_attributes: contentFacetAttributes,
+  } = hostDetails;
+  const contentFacet = propsToCamelCase(contentFacetAttributes);
```

Comment:
When I reload the Errata page I'm getting an error
```
Uncaught Error: propsToCamelCase only takes objects
```

I think this should probably fix it:
```suggestion
  const contentFacet = propsToCamelCase(contentFacetAttributes ?? {});
```